to_do.js

```diff
@@ -0,0 +1,58 @@
+document.addEventListener('DOMContentLoaded', () => {
+
+  // form input from user adds new li element at bottom of ul
+    // pressing submit creates new li element; appends to end of ul
+
+  let to_do = document.querySelector('.to_do');
+  let form = document.querySelector('form');
+  let submit = document.querySelector('#submit_item');
+  let input = document.querySelector('#add_item');
+  // let submit_item = document.
+
+  const addItem = () => {
+    let item = document.createElement('li');
+    item.innerHTML = input.value;
+    to_do.appendChild(item);
+    form.reset();
+  };
+  // if user writes nothing; page shows error message
+  // pressing submit when nothing is entered
+
+  const error = () => {
+    let errorMes = document.createElement('p');
+    if (input.value.length < 1) {
+       let throwErr = errorMes.innerHTML = `Please input a value.`;
+       errorMes.style.color = 'red';
+       form.appendChild(errorMes);
+    }
+  }
```

Comment:
You may want to replace the error message instead of appending a new one every time. This can be done by:
- creating the `p` element, appending it to your form
- adjusting your `error` function using if/else to control the `errorMes` variable's innerText